n2y/converter.py

```diff
@@ -1,18 +1,18 @@
-from collections import deque
+import re
+import logging
+import requests
```

Comment:
See the PEP8 section on module import ordering; I think `requests`, since it's a third party library, should go in the second batch of imports.